themes/drone/layouts/shortcodes/version.html

```diff
@@ -1 +1 @@
-0.7
\ No newline at end of file
+0.8.1
```

Comment:
0.8 instead of 0.8.1

Reply:
I can change that, but 0.8.1 is the latest version and changing that will make users pull an old docker container. What is the intention behind that?

Reply:
`0.8` tag will be updated with all patch releases.  So `0.8` right now **is** `0.8.1` and if `0.8.2` is released then `0.8` will become `0.8.2`.

So it future proofs people to latest `0.8` releases which shouldn't have breaking changes but only bug fixes.

Reply:
Ok, did not know that. Thanks for the explanation. Will adjust the docs now…

Reply:
done